core/src/main/java/hudson/model/AdministrativeMonitor.java

```diff
@@ -36,9 +36,7 @@
 import jenkins.model.Jenkins;
 import org.kohsuke.accmod.Restricted;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
-import org.kohsuke.stapler.StaplerProxy;
-import org.kohsuke.stapler.StaplerRequest;
-import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.*;
```

Comment:
The rest of the code base does not use wild card imports.  Please follow the code style used in the rest of the code.  

The [contributing guide](https://github.com/jenkinsci/jenkins/blob/master/CONTRIBUTING.md#intellij-suggestion) says:

> We also do not recommend * imports in the production code.

```suggestion
import org.kohsuke.stapler.StaplerProxy;
import org.kohsuke.stapler.StaplerRequest;
import org.kohsuke.stapler.StaplerResponse;
```